app/src/ui/publish-repository/publish-private-repository-checker.ts

```diff
@@ -0,0 +1,17 @@
+import { IAPIUserWithPlan } from '../../lib/api'
+import { Account } from '../../models/account'
+
+export function canCreatePrivateRepo(
+  userAccount: Account,
+  targetOrg?: IAPIUserWithPlan | null
```

Comment:
Let's just be explicit in calling this function and not allow `undefined` by making `targetOrg` not optional.